userapi/src/main/java/fk/prof/userapi/Pair.java

```diff
@@ -0,0 +1,40 @@
+package fk.prof.userapi;
```

Comment:
Also, you can move Deserializer class to util package or model.json. A better name for that would be on the lines of ProtoUtils or ProtoDeserializingHelpers

Reply:
Moved Deserializer's methods to ProtoUtils class and Pair.java to util package.